ansible/roles/jenkins-worker/templates/aix.rc2.j2

```diff
@@ -20,9 +20,7 @@ start )
               export HOME=/home/{{server_user}}; \
               export NODE_TEST_DIR=$HOME/tmp; \
               export NODE_COMMON_PIPE="$HOME/test.pipe"; \
-              export OSTYPE=AIX72; \
-              export GIT_SSL_CAINFO="$HOME/ca-bundle.crt"; \
-              export SSL_CERT_FILE="$HOME/ca-bundle.crt"; \
+              export OSTYPE=AIX7; \
```

Comment:
Anyone know what uses `OSTYPE` and whether we need to have a version number in there (as opposed to just `AIX`)?

Reply:
`$(TARNAME)-$(OSTYPE)-$(ARCH).tar.gz` ... so, it may be wrong as it is, but its unrelated to the cert problems, lets leave this cleanup, and the decision about whether its a good idea to later.

Reply:
though also, its initialized with `OSTYPE := $(uname ...` in Makefile, so I'm not sure the env is even used anymore. Either way, its unrelated, so lets not change now.